packages/venia-concept/webpack.config.js

```diff
@@ -13,6 +14,12 @@ module.exports = async env => {
     const unionAndInterfaceTypes = await getUnionAndInterfaceTypes();
 
     const { clientConfig, serviceWorkerConfig } = await configureWebpack({
+        // For Venia only, in the monorepo, always use the sibling packages.
+        alias: {
```

Comment:
This is super ugly tech debt. We can experiment with removing it now that the resolver is refactored.